addons/mail/models/mail_thread.py

```diff
@@ -4699,6 +4699,11 @@ def _thread_to_store(self, store: Store, fields, *, request_list=None):
                 res["suggestedRecipients"] = thread._message_get_suggested_recipients(
                     reply_discussion=True, no_create=True,
                 )
+            if request_list and "contact_fields" in request_list:
+                res["primary_email_field"] = (
+                    self.env[self._name]._mail_get_primary_email_field()
+                )
```

Comment:
Allows easy reading of "res" update
```suggestion
                res["primary_email_field"] = self.env[self._name]._mail_get_primary_email_field()
```